portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -30,29 +35,58 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // If user is logged in and has not set a nickname, redirect to nickname page
     UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      
+    }
```

Comment:
Should something be included here? Or should this be deleted?

Reply:
Forgot to delete. Done!